.gitignore

```diff
@@ -0,0 +1 @@
+AIF360/
```

Comment:
Either .gitignore should include all the files/directories starting with "." (i.e. .DS_Store and .ipynb_checkpoints/) or you should make sure they aren't included in the commit.